docs/serverless/workers/vllm/environment-variables.md

```diff
@@ -0,0 +1,115 @@
+| `DEFAULT_BATCH_SIZE_GROWTH_FACTOR`                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                 | `3`     | `float`      | Growth factor for dynamic batch size.                                                                |
+
+The first request will have a batch size of `DEFAULT_MIN_BATCH_SIZE`, and each subsequent request will have a batch size of `previous_batch_size * DEFAULT_BATCH_SIZE_GROWTH_FACTOR`.
+This will continue until the batch size reaches `DEFAULT_BATCH_SIZE`.
+
+For example, the default values, the batch sizes will be `1, 3, 9, 27, 50, 50, 50, ...`.
+You can also specify this per request, with inputs `max_batch_size`, `min_batch_size`, and `batch_size_growth_factor`. 
+This has nothing to do with vLLM's internal batching, but rather the number of tokens sent in each HTTP request from the worker.
+
+### OpenAI settings
+
+Configure compatibility with OpenAI's API, including output formatting and model naming.
+
+| Name                                | Default     | Type/Choices     | Description                                                                                                                                                                       |
+| ----------------------------------- | ----------- | ---------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `RAW_OPENAI_OUTPUT`                 | `1`         | boolean as `int` | Enables raw OpenAI SSE format string output when streaming. Required to be enabled (which it is by default) for OpenAI compatibility.                                             |
+| `OPENAI_SERVED_MODEL_NAME_OVERRIDE` | `None`      | `str`            | Overrides the name of the served model from model repo/path to specified name, which you will then be able to use the value for the `model` parameter when making OpenAI requests |
```

Comment:
you will then be able to -> you can

Clarify whether you mean
use the value for -> use as the value for/use the value as 
or something else